src/Console/Commands/SubscribeCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Vinelab\Bowler\Console\Commands;
+
+use Illuminate\Console\Command;
+use Vinelab\Bowler\Generators\HandlerGenerator;
+
+/**
+ * @author Kinane Domloje <kinane@vinelab.com>
+ */
+class SubscribeCommand extends Command
+{
+    const TYPE = 'subscribe';
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $signature = 'bowler:subscribe
```

Comment:
using verbs such as `subscribe` gives the impression that it will "run" something, such as `consume`. But when it comes to generators, it's better to use the `make` as means of consistency with Laravel (i.e. `bowler:make:queue` and `bowler:make:subscription` or a better name you can think of.